go/tools/builders/BUILD.bazel

```diff
@@ -23,6 +23,24 @@ go_test(
     ],
 )
 
+go_test(
+    name = "stdliblist_test",
+    size = "small",
+    srcs = [
+        "env.go",
+        "flags.go",
+        "replicate.go",
+        "stdliblist.go",
+        "stdliblist_test.go",
+    ],
+    data = ["@go_sdk//:files"],
+    # -sdk in stdliblist needs to be a relative path, otherwise it breaks
+    # assumptions of cloning go_sdk, thus we need to set up the test in a
+    # way that go_sdk is under the directory where test is run.
+    rundir = "..",
```

Comment:
this is an easy way to avoid having to break assumption of go_sdk to be absolute path.
by setting it this way, the structure will become
```
/a/b/c/runfiles/   <- this will be where the test is run with rundir being ..
    go_sdk         <- this will be where the data files are
    workspace      <- this will be where the test is run with rundir being . 
```
then naturally `go_sdk` is the relative path.